src/main/c/src/SerialImp.c

```diff
@@ -4380,7 +4379,7 @@ JNIEXPORT void JNICALL RXTXPort(eventLoop)( JNIEnv *env, jobject jobj )
 			/* nothing goes between this call and select */
 			if( eis.closing )
 			{
-				report("eventLoop: got interrupt\n");
+				report_verbose("eventLoop interrupted");
```

Comment:
I think the newline character is required here, right?

Reply:
Why do you mean?

Reply:
Never mind, I got confused. I thought the format string needed a trailing newline for some reason. Forgot that it's the opposite, and that went to extra lengths to strip trailing newlines in the SLF4J wrapper.